Add empty test structure for MessageFramework

mnurzyns opened the pull request:
To create tests for Message class we need to have structure
with TestCore due to create fake (mock) messages and we also
want to prepare some space to test it.

Commit:
Add empty test structure for MessageFramework

To create tests for Message class we need to have structure
with TestCore due to create fake (mock) messages and we also
want to prepare some space to test it.

Commit:
Empty file for tests for StringSum

StringSum is easy and small class so it's not need full
TestCore, simple test file will be enought.

Commit:
Remove unnecessary includes

Due to copy-paste flow there were commited not needed includes from
ChatRequest.

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/CMakeLists.txt:
```diff
@@ -0,0 +1,36 @@
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTestCore.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTest.cpp PARENT_SCOPE)
+
+#SOURCES FILES WHICH ARE NEEDED TO COMPILE TESTS
```
Do wyrzucenia, to jest tylko potrzebne (nie wiem czemu) tylko w tym jednym CMakeu z którego to kopiowałes

mnurzyns replied:
Tak, ale tprzyszłościow chcielibyśmy by CMake dobierał tylko te mu potrzebne, więc w każdym pliku będzie pojedyńcze dodanie tego. W takim razie uważam, żeby zostawić

marcinwozniak0 replied:
Wyrzuć to, aktualnie nam to nie potrzebne a jest tylko śmietnik w tmy CMaku

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/MessageTestCore.hpp:
```diff
@@ -0,0 +1,18 @@
+
+#include <gtest/gtest.h>
+
+#include <ChatRequest.hpp>
```
wszystkie includy zbedne

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/MessageTestCore.cpp:
```diff
@@ -0,0 +1,28 @@
+#include <thread>
+
+#include <MessageTestCore.hpp>
+#include <SignIn.hpp>
```
zbedne includy

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/MessageTest.cpp:
```diff
@@ -0,0 +1,7 @@
+#include <MessageTestCore.hpp>
+
+
```
zbedny enter

mnurzyns replied:
> struktura plików, która się kompiluje
> czepianie się entera

Dobrze, że nie przyjebałeś się do mało legitnego testu XD

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/CMakeLists.txt:
```diff
@@ -0,0 +1,36 @@
+    ${CMAKE_CURRENT_SOURCE_DIR}/StringSumTest.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTest.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTestCore.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTest.cpp PARENT_SCOPE)
```
Zdublowany MessageTest

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/MessageTestCore.cpp:
```diff
@@ -1,10 +1,6 @@
-#include <SignOut.hpp>
-#include <RegisterUser.hpp>
-#include <GlobalVariables.hpp>
+#include <MessageTestCore.hpp>S
```
weź kompiluj to lokalnie, bo coś mi się wydaje że ta linijka zespuje CI :)

Commit:
Add TestCore for PurgeMessage class

Similar to Message class PurgeMessage is a little bit big
and need testCore, because for example it need strings, messages
and purgeMessage which are fake.

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/PurgeMessageTestCore.cpp:
```diff
@@ -0,0 +1,24 @@
+#include <thread>
+
+#include <PurgeMessageTestCore.hpp>
+
```
enter

Commit:
Remove not needed blank lines and CMake source set.

After review ew decided to delete CMake source set due to
include it in other test CMakeLists.txt. Also there was delete
blank lines.

Commit:
Remove test falseIsNotTrue which fail

This test should fail because it was only added to check
is EXPECT_TRUE(false) will give as FAIL when we run UT.
Now it should be deleted due to running UT on CI.
